docs/api-gateway-v2/03-04-migration.md

```diff
@@ -0,0 +1,197 @@
+
+
+>**CAUTION:** Some API configurations are too complex or do not meet all requirements to pass the automatic migration. To ensure all your services are exposed using APIRules, [verify the migration outcome](#verify-automatic-migration). If there still are API resources in use, follow [this](#manual-migration) guide to migrate them.
+
+
+Skipping some resources during migration will not break the way existing services are exposed. However, bear in mind that if you want to introduce changes or remove the API resource, it won't have any effect on how the service is exposed because the service will still use the original configuration. 
+
+
+## Verify automatic migration
+
+Follow these steps to verify if all the API resources were migrated to APIRules:
+
+1. Once the Kyma upgrade finishes, list the existing API objects:
+
+    ```shell script
+      kubectl get apis --all-namespaces
+    ```
+
+2. As API objects are removed after a successful migration, the list shows all resources that the migration job skipped. Fetch the logs from the job's Pod to learn the reasons for this behavior: 
+
+    ```shell script
+    kubectl logs api-gateway-api-migrator-job -n kyma-system
+    ```
+
+>**NOTE:** If the migration process skipped the API resources due to the invalid status or a blacklisted label, you either should not or will not be able to migrate them.
+
+## Manual migration
+
+This guide shows how you can manually migrate API resources to API Rules.
+
+>**NOTE:** Before you start the manual migration process, see the [API CR](/components/api-gateway/#custom-resource-api-sample-custom-resource) and [APIRule CR](/components/api-gateway-v2#custom-resource-api-rule) documents for the custom resource detailed definition.
+
+Follow these steps:
+
+1. Fetch the API resource you want to migrate:
+
+    ```shell script
+    kubectl get api {NAME} -n {NAMESPACE} -o yaml
+    ```
+
+2. Create an APIRule resource based on the API object's specification:
+
+>**NOTE:** This step focuses on the`spec` part of the CRD. You can `metadata` field in a preferred way and do not copy the [`status`](components/api-gateway-v2#custom-resource-api-rule-additional-information) field from the original API resource.
+
+a. Replace the default value (`kyma-gateway.kyma-system.svc.cluster.local`) for the **gateway** parameter with Istio Gateway used to expose services. 
+
+b. Copy the values for **service.name** and **service.port** parameters from the corresponding fields of the API object. 
+
+c. Set the value for **service.host** parameter to any temporary value which includes the domain. For example, if the value for the **hostname** parameter of the API object was set to `sample-service.kyma.local`, change it to `temp-sample-service.kyma.local` so that differs from the original value. Make sure other services on your cluster do not use this hostname. 
+
+d. Configure the **rules** parameter of APIRule.
```

Comment:
I was wondering - maybe we can have an example with such a configuration here - I mean we have this big section explaining all the differences and comparing API but maybe an example would be good?